README.md

```diff
@@ -154,16 +154,26 @@ Valkyrie.config.metadata_adapter.query_service.find_all_of_model(model: MyModel)
 ## Installing a Development environment
 
 ### External Requirements
-* PostgreSQL with the uuid-ossp extension.
-  * Note: Enabling uuid-ossp requires database superuser privileges.
-    * From `psql`: `alter user [username] with superuser;`
-
-### To run the test suite
-1. Start Solr and Fedora servers for testing with `rake server:test`
-1. Run `rake db:create` (First time only)
-1. Run `rake db:migrate`
+* [Docker](https://store.docker.com/search?offering=community&type=edition) version >= 17.09.0
+
+### Starting the development mode dependencies
+1. Start Solr, Fedora, and PostgreSQL with `rake docker:dev:daemon` (or `rake docker:dev:up` in a separate shell to run them in the foreground)
```

Comment:
When I run `rake docker:dev:up`, I get an error message:

```
Couldn't connect to Docker daemon - you might need to run `docker-machine start default`.
rake aborted!
Docker::Compose::Error: 'up' failed with status 'pid 21346 exit 1': Couldn't connect to Docker daemon - you might need to run `docker-machine start default`.
/Users/escowles/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/Users/escowles/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Tasks: TOP => docker:dev:up
(See full trace by running task with --trace)
```

Is there something I need to start before running the rake task?

Reply:
If you installed the official Mac release, you might need to run the Docker app. Your menu bar should have this little guy in it:

![item-0 2018-03-27 13-32-39](https://user-images.githubusercontent.com/196872/37993376-5ac754be-31c3-11e8-9200-02a9b4cf005c.png)